python/erdos/streams.py

```diff
@@ -180,22 +194,9 @@ class IngestStream(object):
     :py:func:`IngestStream.send` to enable the driver to send data to the
     operator to which it was connected.
     """
-    def __init__(self, _name: Union[str, None] = None):
-        self._py_ingest_stream = PyIngestStream(_name)
 
-    @property
-    def name(self) -> str:
-        """ The name of the stream. The stream ID if none was given."""
-        return self._py_ingest_stream.name()
-
-    @name.setter
-    def name(self, name: str):
-        self._py_ingest_stream.set_name(name)
-
-    @property
-    def id(self) -> str:
-        """ The id of the IngestStream. """
-        return uuid.UUID(self._py_ingest_stream.id())
+    def __init__(self, name: Union[str, None] = None):
+        super().__init__(PyIngestStream(name))
```

Comment:
Could you rebase on the latest commit? I'm a little worried by these changes since they appear to be duplicates of commits that already occured. E.g. `IngestStream` should inherit from `Stream`, but that doesn't appear to be the case in this file.

Reply:
Done.